base/actions/actions.py

```diff
@@ -87,7 +87,6 @@ def fractional_part(f):
             f"{fractional_part(latitude)}"
             f"{int(longitude):+04d}"
             f"{fractional_part(longitude)}"
-            "/"
```

Comment:
We should just make sure this works downstream, healthcheck django app + the dashboards etc